data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -0,0 +1,113 @@
+# Suffixes
```

Comment:
I know we had talked about maybe having constants set on the ETL class, but I like this approach a lot better. The fields get referenced and handled in lots of places so it makes sense to have them created here! Nice work.